forge-gui/res/cardsfolder/c/cloudform.txt

```diff
@@ -3,8 +3,8 @@ ManaCost:1 U U
 Types:Enchantment
 T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigManifest | TriggerDescription$ When CARDNAME enters, it becomes an Aura with enchant creature. Manifest the top card of your library and attach CARDNAME to it. (To manifest a card, put it onto the battlefield face down as a 2/2 creature. You may turn it face up at any time for its mana cost if it's a creature card.)
 SVar:TrigManifest:DB$ Manifest | RememberManifested$ True | SubAbility$ Aurify
-SVar:Aurify:DB$ Animate | IsPresent$ Card.Self | Types$ Aura | OverwriteSpells$ True | Abilities$ NewAttach | Keywords$ Enchant creature | Duration$ Permanent | SubAbility$ RageformAttach
+SVar:Aurify:DB$ Animate | Types$ Aura | OverwriteSpells$ True | Abilities$ NewAttach | Keywords$ Enchant creature | Duration$ Permanent | SubAbility$ RageformAttach
```

Comment:
What happens if the Enchantment gets Destroyed/Removed in response to the Trigger? Does it try to attach it from the Graveyard?

Reply:
no, it will get skipped from failing timestamp check